sourcegraph/cody
# Enable manually editing search context

Adds a button to ContextCell that copies the fetched context items to the previous input as context chips.

https://github.com/user-attachments/assets/d3c24e37-d3b9-48a4-94fd-11090c78a9d9

Outstanding todos:
- [x] Change text from "manually edit context" to "edit context"
- [x] Make work when search intent is detected. Text should be "move results to input"
- [x] Expand search results by default


## Test plan

- Use an @repo chip
- Click the "Edit context manually" button
- Verify context items are added to the input
